src/Models/Concerns/HasDefaultRelations.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace OwowAgency\Gossip\Models\Concerns;
+
+trait HasDefaultRelations
+{
+    /**
+     * Scope a query to only include popular users.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  array  $args
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeWithRelations($query, ...$args)
```

Comment:
Is "[Eager loading by default](https://laravel.com/docs/8.x/eloquent-relationships#eager-loading-by-default)" what you need?

Also why not call it `scopeWithDefaultRelations`?

Reply:
I've check eager loading by default. But I've parameters, so that's why I couldn't use it.